eng/scripts/automation/GenerateAndBuildLib.ps1

```diff
@@ -753,7 +753,7 @@ function GeneratePackage()
     } else {
         # Build
         Write-Host "Start to build sdk: $projectFolder"
-        dotnet build $projectFolder
+        dotnet build $projectFolder /p:RunApiCompat=$false
```

Comment:
Why did we disable this? Part of the point of running during SDK automation is to find API breaking changes.

Reply:
This ties to the work of making SDK automation required  in swagger PR review.
Right now, .NET Track 2 SDK automation is failing mostly due to breaking changes.
We do not deal with the breaking changes until the SDK generation, so it seems no need to fail it during swagger PR review.
cc @ArthurMa1978

Reply:
Yes, we don't want to run the apicompat check at this job which blocks the swagger review process.
Next step is add an optional job for apicompat checking.

Reply:
How does it block the swagger review process? Even if the check is red that is a break and should be handled but it shouldn't bock the swagger review. We should just approve the break fix it. I think this is a bad idea to remove this completely. If there are some specific libraries that we want to disable this for then we should do it in the project instead of disabling for everyone.